VERSIONING.md

```diff
@@ -15,8 +15,10 @@ In certain case it's also possible for the `Dockerfile` itself to change without
 
 Follow these steps when an update is ready to be pushed to Docker Hub:
 
+- Make sure you are on the master branch and that the HEAD is pointing to the lastest commit (`git checkout master`). `git log --name-status HEAD^..HEAD` should give you the same commit number that the latest commit you see at <https://github.com/mailserver2/mailserver>
 - Tag the current GitHub commit in the `mailserver2` repo with the next version, e.g. `git tag -a v1.0.1 -m "update postfixadmin to version 1.0.1, ... other changes here"`
-- Build the new image
+- Push tags with `git push --tags`
```

Comment:
Maybe add that we should also create a new github release (https://github.com/mailserver2/mailserver/releases) after pushing the tag. If we don't the latest release that github shows will be an old one. That will be confusing.

Reply:
Added that